courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb

```diff
@@ -19,7 +19,7 @@
     "\n",
```

Comment:
<p>Please add two spaces before the "Introduction" header?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Updated the same in the notebook.